packages/react-composites/src/composites/CallWithChatComposite/Strings.tsx

```diff
@@ -37,6 +37,12 @@ export interface CallWithChatCompositeStrings {
    * {@link CallWithChatComposite} control bar Chat button enhanced tooltip string.
    */
   chatButtonTooltipClosedWithMessageCount: string;
+  /**
+   * Title for the audio device selection sub-menu in more buttton drawer.
```

Comment:
nit: remove `in more buttton drawer` as this is currently used in the local preview context menu (which really _should_ be a more drawer but currently isn't)